container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -290,8 +343,8 @@ function generate_test_results {
 
 export OS_CLOUD=default
 
-if [[ ! -z ${TEMPEST_NEUTRON_IMAGE_URL} ]]; then
-    generate_extra_tempest_configuration
+if [[ ! ${#TEMPEST_EXTRA_IMAGES_NAME[@]} -eq 0 ]]; then
```

Comment:
what if TEMPEST_EXTRA_IMAGES_NAME is not set, does it work?

I tried this:
```
$ cat sh.sh 
TEMPEST_EXTRA_IMAGES_NAME="${TEMPEST_EXTRA_IMAGES_NAME:-}"

if [[ ! ${#TEMPEST_EXTRA_IMAGES_NAME[@]} -eq 0 ]]; then
    echo "image handling"
fi
```

and it printed the echo - it got inside the if and it shouldn't
```
$ sh sh.sh 
image handling
```

Reply:
In the example, there is conversion to an array missing (line 143). This works:

```
TEMPEST_EXTRA_IMAGES_NAME="${TEMPEST_EXTRA_IMAGES_NAME:-}"
read -ra TEMPEST_EXTRA_IMAGES_NAME <<< $TEMPEST_EXTRA_IMAGES_NAME

if [[ ! ${#TEMPEST_EXTRA_IMAGES_NAME[@]} -eq 0 ]]; then
    echo "image handling"
fi
```

This works ^^.